src/components/PopupWithForm.js

```diff
@@ -28,4 +30,23 @@ export default class PopupWithForm extends Popup {
     this._popupForm.reset();
     super.close();
   }
+
+  setLoading(isLoading) {
+    if (isLoading) {
+      this._submitButton.textContent = "loading...";
+    } else {
+      this._submitButton.textContent = this._submitButtonText;
+    }
+  }
+
+  renderLoading() {
+    const img = document.createElement("img");
+    img.src = imageSrc;
+    return img;
+  }
+
+  loadImage() {
```

Comment:
Needs correction: please remove this method, you don't need it here

Reply:
I removed the "loadImage" method.